### ISSUE TITLE
WCAG 1.3.1,WCAG 3.3.2: Form elements must have labels (input[type="checkbox"])

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2277]
#### Issue
`Form elements must have labels` ([label](https://dequeuniversity.com/rules/axe/3.2/label?application=msftAI))
#### Target application
[Microsoft MakeCode for micro:bit](https://makecode.microbit.org/#editor)
#### Element path
.field:nth-child(2) > .checkbox.toggle.ui > input[type="checkbox"]
#### Snippet
`<input type="checkbox" aria-checked="false">`
#### How to fix
    Fix any of the following:
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Form element does not have an implicit (wrapped) <label>
      Form element does not have an explicit <label>
      Element has no title attribute or the title attribute is empty
#### Environment
Chrome version 75.0.3770.142
This accessibility issue was found using Accessibility Insights for Web 2.7.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

![Screen Shot 2019-07-17 at 5 26 25 PM](https://user-images.githubusercontent.com/5615930/61420596-09994800-a8b8-11e9-9484-4fbe6a705e4d.png)
